plugins/eos-blacklist/gs-plugin-eos-blacklist.c

```diff
@@ -207,17 +215,43 @@ app_get_flatpak_ref_name (GsApp *app)
 	return gs_app_get_metadata_item (app, "flatpak::RefName");
 }
 
+static gchar **
+get_default_locales (GsPlugin *plugin)
```

Comment:
```suggestion
get_flatpak_default_locales (GsPlugin *plugin)
```
(to make it a bit clearer that this gets the preferences from flatpak, rather than the environment, or anywhere else)